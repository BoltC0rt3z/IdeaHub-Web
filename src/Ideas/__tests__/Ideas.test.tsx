import React from "react";
import { mount, ReactWrapper } from "enzyme";

import { Ideas } from "..";

describe("Tests for Ideas", () => {
  const wrapper: ReactWrapper = mount(<Ideas />);

  it("Should render ideas without crushing", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
