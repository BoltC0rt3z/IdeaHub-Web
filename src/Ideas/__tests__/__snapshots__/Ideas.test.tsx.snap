// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tests for Ideas Should render ideas without crushing 1`] = `ReactWrapper {}`;
